src/styles/main.scss

```diff
@@ -0,0 +1,65 @@
+@import './utils/variables';
+
+body {
+  margin: 0;
+  padding-top: 25px;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: $col-count * $cell-width + $gap * $col-count;
```

Comment:
Ensure that the variables `$col-count`, `$cell-width`, and `$gap` are defined in the imported variables file. This line uses these variables for calculating the width, and they must be correctly set for the layout to work as expected.